smokey/src/main/java/frc/robot/commands/AllignWithTag.java

```diff
@@ -0,0 +1,83 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: AllignWithTag.java
+// Intent: Forms a command to allign itself with a designated april tag.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.common.MotorUtils;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class AllignWithTag extends CommandBase{
+    private boolean done = false;
+    private double tagID;
+    private double velocityValue = 0.3;
+    private PIDController yPID = new PIDController(1.0, 0.0, 0.0);
```

Comment:
So I will point you at camel casing and the published Java naming conventions that Oracle has at: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html .  While some might argue that acronyms like PID are super common (and therefore can be used), I think just as many would prefer to see the long form of the name (ProportionalIntegralDerivative) would be appropriate.  Either way the ID in PID here breaks camel casing rules.  So the name should either be yPid or yProportionalIntegralDerivative or even better yAxisProportionalIntegralDerivative.

Reply:
changed the name.